docker/patch/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM scratch
+LABEL image="ghcr.io/labring/lvscare:v1.1.3-beta.8"
+LABEL sealos.io.type="patch"
+COPY images ./images
+COPY registry ./registry
```

Comment:
I wonder where `./registry`, `./opt` and `./cri` are. I only see `./images`.

Reply:
registry , opt and cri dir in basic image